composer.json

```diff
@@ -10,7 +10,7 @@
     },
     "require-dev": {
 				"phpunit/phpunit": "^4.8 || ^5.0",
-				"dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
+				"dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
```

Comment:
I'm curious about this line - is this a required change for this PR to work?

Reply:
From the PR description:

>Make sure #1392 is merged before this PR to bring the composer v2 updates to master.

In hindsight I should've just not commited this, sorry about that.

We can safely drop the composer changes out of the PR, but alternatively we could merge #1392 and this diff will go away.